modules/reference/pages/systems/system-details/sd-migrate.adoc

```diff
@@ -1,12 +1,13 @@
 [[sd-migrate]]
-= SD Migrate
+= Migrate
 
-This subtab provides the option to migrate systems between organizations.
-Select an organization form the dropdown [guimenu]``Migrate System Between Organizations`` and click btn:[Migrate System] to initiate the migration.
+The menu:Systems[Details > Migrate] section allows you to move clients between organizations.
```

Comment:
I'm not too happy about this menu path.  The "Systems" section does not have a Details tab with a Migrate subtab.
In the past, we used the term "System Details" page as the root of the tabs and subtabs of a system/client. This might also be confusing.

Reply:
I sort of figured it was OK on the page where we discuss that subtab directly. I wouldn't use it anywhere else, though, I'd say "navigate to menu:System[System List], select the client, navigate to the menu:Details[Migrate] subtab" (or whatever).